tools/Snippets/EditorConfigExcludedSymbolNamesTest.snippet

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
```

Comment:
@dotpaul I will ask you how to use this snippet next time we meet in person :)

Reply:
https://docs.microsoft.com/visualstudio/ide/walkthrough-creating-a-code-snippet

This Visual Studio thing is pretty neat!